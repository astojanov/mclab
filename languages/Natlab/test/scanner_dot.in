a.b
d.e.f
g . h
i. ...
j
k.1