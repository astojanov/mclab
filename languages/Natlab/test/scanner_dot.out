IDENTIFIER 1 1 1 a
DOT 1 2 1
IDENTIFIER 1 3 1 b
LINE_TERMINATOR 1 4 1
IDENTIFIER 2 1 1 d
DOT 2 2 1
IDENTIFIER 2 3 1 e
DOT 2 4 1
IDENTIFIER 2 5 1 f
LINE_TERMINATOR 2 6 1
IDENTIFIER 3 1 1 g
DOT 3 3 1
IDENTIFIER 3 5 1 h
LINE_TERMINATOR 3 6 1
IDENTIFIER 4 1 1 i
DOT 4 2 1
IDENTIFIER 5 1 1 j
LINE_TERMINATOR 5 2 1
IDENTIFIER 6 1 1 k
NUMBER 6 2 2 .1
